presto-iceberg/src/main/java/com/facebook/presto/iceberg/PartitionFields.java

```diff
@@ -190,4 +197,30 @@ private static String toPartitionField(PartitionSpec spec, PartitionField field)
 
         throw new UnsupportedOperationException("Unsupported partition transform: " + field);
     }
+
+    public static String quotedName(String name)
+    {
+        if (UNQUOTED_IDENTIFIER_PATTERN.matcher(name).matches()) {
+            return name;
+        }
+        return '"' + name.replace("\"", "\"\"") + '"';
+    }
+
+    public static String fromIdentifierToColumn(String identifier)
+    {
+        if (QUOTED_IDENTIFIER_PATTERN.matcher(identifier).matches()) {
+            // We only support lowercase quoted identifiers for now.
+            // See https://github.com/trinodb/trino/issues/12226#issuecomment-1128839259
+            // TODO: Enhance quoted identifiers support in Iceberg partitioning to support mixed case identifiers
+            //  See https://github.com/trinodb/trino/issues/12668
```

Comment:
Can you help more clearly describe the limitation of this comment? Does this mean that identifiers in the CREATE TABLE specifying the sort order have to be quoted and lowercase?

We should clearly document this limitation to users.

Reply:
Yes @ZacBlanco, It only supports lowercase quoted identifiers for now; otherwise, it throws an error, as shown in the following screenshot.

<img width="944" alt="image" src="https://github.com/user-attachments/assets/1f0a0e50-4cd5-4fea-831c-66ef9743cb97" />

Reply:
We should document this limitation and provide a test that asserts the failure

Reply:
Should we keep the handling logic of the sort order column consistent with that of the table column? For example, when we create a table like this:

```
create table test("ID" int);
```

We will succeed with the column name lowercased to `id`. Is there a reason that the sort by column could not be handled like this?